src/runtime/test/particle-api-test.ts

```diff
@@ -14,8 +14,13 @@ import {StubLoader} from '../testing/stub-loader.js';
 import {TestHelper} from '../testing/test-helper.js';
 import * as util from '../testing/test-util.js';
 import {Arc} from '../arc.js';
+import {IdGenerator} from '../id.js';
+import {Manifest} from '../manifest.js';
+import {Schema} from '../schema.js';
+import {EntityType} from '../type.js';
+import {VariableStore} from '../store.js';
 
-async function loadFilesIntoNewArc(fileMap) {
+async function loadFilesIntoNewArc(fileMap: {[index:string]: string, manifest: string}) {
```

Comment:
This seems like a good use case for `Dictionary<string>`. See `src/runtime/hot.ts`. 

Specifically, this should be `Dictionary<string> & {manifest: string}`.